spec/resources/pseudo/style.css

```diff
@@ -13,6 +13,6 @@
 
 #dom-node {
     background-color: white;
-    font-family: monospace;
+    font-family: Arial;
```

Comment:
this can be a problem for non-Windows users, I'm not sure how it will work for my Linux workstation, I don't have MS fonts installed by default. Can it be just "sans-serif" or sth like that?